### ISSUE TITLE
Error on boot with sharp (Mac Sierra, Node v8.9.0)

### ISSUE THREAD

**Issue by niftylettuce** [lovell/sharp#1043]
Related to #939 and #1031 

Node v8.9.0 with fresh `sharp` install on Mac Sierra related to libvips install issue:

```sh
module.js:664
  return process.dlopen(module, path._makeLong(filename));
                 ^

Error: dlopen(/Users/user/Projects/project/template/node_modules/sharp/build/Release/sharp.node, 1): Library not loaded: @rpath/libvips-cpp.42.dylib
  Referenced from: /Users/user/Projects/project/template/node_modules/sharp/build/Release/sharp.node
  Reason: image not found
    at Object.Module._extensions..node (module.js:664:18)
    at Module.load (module.js:554:32)
    at tryModuleLoad (module.js:497:12)
    at Function.Module._load (module.js:489:3)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/user/Projects/project/template/node_modules/sharp/lib/constructor.js:9:15)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32)
    at tryModuleLoad (module.js:497:12)
    at Function.Module._load (module.js:489:3)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/user/Projects/project/template/node_modules/sharp/lib/index.js:3:15)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32)
    at tryModuleLoad (module.js:497:12)
    at Function.Module._load (module.js:489:3)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
```

**Comment by lovell** [comment]
* Are there any errors during `npm install sharp`?
* Does the `node_modules/sharp/vendor` directory, created during `npm install`, exist?
* Has `vips` been installed globally on this machine via `brew` at any time in the past?
* What does `otool -L node_modules/sharp/build/Release/sharp.node` say?

**Comment by lovell** [comment]
@niftylettuce Are you able to provide the requested details should this problem still be occurring?

**Comment by davidbarton** [comment]
Had exact same error. Fixed by upgrade from `"^0.18.4"` to `"^0.20.0"`

**Comment by owenhoskins** [comment]
This error randomly occurred for me just now after adding an unrelated package with yarn. I was already at `"^0.20.0"` but running `yarn upgrade sharp` resolved the issue anyway!
